docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -308,3 +254,88 @@ Then, open http://localhost:8080/hello/quarkus in a browser, and you should get
 
 Like any other Quarkus applications, you can package it with: `mvn package`.
 You can also package the application into a native executable with: `mvn package -Pnative`.
+
+== Generating Java files from proto with protobuf-maven-plugin
+
+Alternatively to using Quarkus code generation to generate stubs for `proto` files, you can also use
+`protobuf-maven-plugin` for this purpose.
+To do it, edit the `pom.xml` file to add the quarkus gRPC extension dependency and the JSR 305 (just under `<dependencies>`):
```

Comment:
```suggestion
To do it, edit the `pom.xml` file to add the Quarkus gRPC extension dependency and the JSR 305 (just under `<dependencies>`):
```

Reply:
You're talking about the JSR 305 but it's not in the extract below?

Reply:
I only moved the section in the original commit, it seems it was a bit outdated.
I removed JSR305 and did some further fixes to the docs.